data-serving/scripts/aggregate/aggregate/app.py

```diff
@@ -141,13 +144,15 @@ def get_jhu_counts():
     )
     jhu_counts = jhu_counts.set_index("Country_Region")
     jhu_counts = pd.Series(jhu_counts.values.flatten(), index=jhu_counts.index)
+    logging.debug("Got aggregated counts from JHU")
```

Comment:
This line seems like it's at the same level of importance as the two above, which have `info` severity.

Reply:
Updated.